docs/Home.md

```diff
@@ -0,0 +1,37 @@
+# About starterkit-mobile-application-flutter
+
+ERNI Academy mobile boilerplate to start a cross-platform Flutter mobile application.
+
+ [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/ERNI-Academy/starterkit-mobile-application-flutter/badge)](https://securityscorecards.dev/viewer/?uri=github.com/ERNI-Academy/starterkit-mobile-application-flutter) [![codecov](https://codecov.io/gh/ERNI-Academy/starterkit-mobile-application-flutter/graph/badge.svg?token=IQPAQ95HC6)](https://codecov.io/gh/ERNI-Academy/starterkit-mobile-application-flutter) [![Code Analysis](https://github.com/ERNI-Academy/starterkit-mobile-application-flutter/actions/workflows/ci-code-analysis.yml/badge.svg)](https://github.com/ERNI-Academy/starterkit-mobile-application-flutter/actions/workflows/ci-code-analysis.yml) [![Android Build](https://github.com/ERNI-Academy/starterkit-mobile-application-flutter/actions/workflows/ci-android.yml/badge.svg)](https://github.com/ERNI-Academy/starterkit-mobile-application-flutter/actions/workflows/ci-android.yml) [![iOS Build](https://github.com/ERNI-Academy/starterkit-mobile-application-flutter/actions/workflows/ci-ios.yml/badge.svg)](https://github.com/ERNI-Academy/starterkit-mobile-application-flutter/actions/workflows/ci-ios.yml)
+
+## Built With
+
+- [Flutter](https://flutter.dev)
+
+## Getting Started
+
+- [Install Flutter](https://docs.flutter.dev/get-started/install)
+- [Introduction to Dart Language](https://dart.dev/guides/language/language-tour)
+- [Flutter Documentation](https://docs.flutter.dev/)
+
+## Prerequisites
+
+**Flutter**
+- v3.13.5+ on stable channel
```

Comment:
Just a thought here: Refering to a specific version will be outdated quite soon, especially for a starterkit application that is not constantly being maintained...
I suggest to ask the reader to install the latest stable version. Then if that causes trouble to open a new issue here. That way the starterkit will more likely be kept up to date, as we'll be getting feedback from users.